build/installers/scripts/linux/studio-installer-checker-linux.sh

```diff
@@ -0,0 +1,96 @@
+source ../config/env.properties 
```

Comment:
lets put this until `build/installers/tests`

Reply:
Moving all scripts into build/installers/tests folder.